app/assets/stylesheets/application.scss

```diff
@@ -52,3 +52,23 @@ a.btn.disabled[title] {
 #command_command {
   font-family: monospace;
 }
+
+table td span {
+  &.additions {
+    color: green;
+
+    &::before {
+      content: "+";
+      padding: 0 2px;
+    }
+  }
+
+  &.deletions {
+    color: red;
+
+    &::before {
+      content: "-";
+      padding: 0 2px;
+    }
+  }
```

Comment:
If it's specific to deploys (seems to be) it could be placed in `deploys.scss`

Reply:
The code is actually shared between both `deploys` and `releases`. I've moved it in the global.css which already has a `.label` definition